### ISSUE TITLE
White screen after login with SystanciaGate

### ISSUE THREAD

**Issue by Jayddee23** [Ylianst/MeshCentral#5846]
Hello,
We have updated our SystanciaGate portal (IPDiva) and Mesh server from version 1.1.12 to 1.1.21.
Since then, the web resource to meshcentral that we had set up for our service providers no longer works.
When you click on it, you get the login screen, but after validating your login/password to establish the connection, the window goes blank.
I tried to inspect the page and found that the body and <div> are all still in "display: none".
If I remove this "display: none", I can see the menu, but I can't access anything except the bottom right "CGU & Vie Privée", which links to the page.

![image](https://github.com/Ylianst/MeshCentral/assets/101196731/380f9758-9073-495c-a808-ee5b6a8a841f)

Internally, without going through SystanciaGate, it works fine.
So we can't determine whether the problem stems from a SystanciaGate settings issue or whether something new in the latest mesh versions is being misinterpreted.

Any ideas?

Thank you,
Best regards,

Translated with DeepL.com (free version)

**Comment by si458** [comment]
**you didnt follow the bug template report, you are missing key information we need to help you**

can you share your config.json? (hide secret info)
whats the OS your running meshcentral on?
whats the nodejs version you are running?
have you checked the console tab when you inspect the web page in your browser to see what errors are listed?

**Comment by si458** [comment]
i noticed in the first screenshot it says `lang="fr"`, 
can you try access the webpage using `https://mymeshcentraldomain.com/?lang=en` ?
just want to rule out a translation issue with the handlebars files

**Comment by Jayddee23** [comment]
Yes I came across another post before opening this one that mentioned this alternative, logically I don't put the language at the end of the url, so I tried adding "?lang=fr" or "?lang=en" but same problem.

**Comment by Jayddee23** [comment]
I think so too, I'll get closer to them, thanks for taking the time to look anyway.
